wizard/src/types.ts

```diff
@@ -69,24 +70,29 @@ export interface CsvData {
   // array should be present as keys of columnValuesSampled.
   // Note that the length of all column-values need not be the same, e.g. due to
   // the removal of duplicate values.
-  columnValuesSampled: Map<string, Array<string>>,
+  columnValuesSampled: Map<string, Array<string>>;
 
   // rowsForDisplay is a mapping from the row index in the original csv file to
   // the contents of the row. This is a convenience structure to assist with
   // previews etc. It is not expected to contain the entire csv data, i.e. there
   // is no expectation that this structure contains all rows.
   // It is also assumed that order of values in the array will correspond to
   // the orderedColumnNames.
-  rowsForDisplay: Map<BigInt, Array<string>>,
-}
+  rowsForDisplay: Map<BigInt, Array<string>>;
+
+  // if csv input was a user uploaded file, the uploaded csv file.
+  rawCsvFile?: File;
 
+  // if csv input was a user entered url, the url to get the csv file.
+  rawCsvUrl?: string;
```

Comment:
nit: one of these must be set?

Reply:
I don't think there's a way to enforce that one of these is set, but I can add a comment

Reply:
Yup, comment is good.

Reply:
done